### PR TITLE
Adding main.swift to native swift template

### PR DESCRIPTION
**Pull request by wmathurin**
Without it, pin code will crash app

### CODE REVIEW

File: SalesforceMobileSDK.xcworkspace/xcshareddata/xcschemes/Everything.xcscheme [hunk 1]
```diff
@@ -328,6 +328,90 @@
...
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "CEA883ED1C19160E008D871B"
+               BuildableName = "libSmartSync.a"
+               BlueprintName = "SmartSyncStatic"
+               ReferencedContainer = "container:libs/SmartSync/SmartSync.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
```
Comment (wmathurin) [inline SalesforceMobileSDK.xcworkspace/xcshareddata/xcschemes/Everything.xcscheme]: Unrelated change

**Comment by bhariharan** [comment]
@wmathurin Do you want to do this against ```master```, so that it'll de facto make it into our patch release?

**Comment by khawkins** [comment]
Seems fine to me.

**Comment by wmathurin** [comment]
Will resubmit against master
